discover/types.go

```diff
@@ -183,3 +183,17 @@ func (si SystemInfo) GetOptimalThreadCount() int {
 
 	return coreCount
 }
+
+// For each GPU, check if it does NOT support flash attention
+func (l GpuInfoList) FlashAttentionSupported() bool {
```

Comment:
I believe we can omit this change and keep the lookup below in server.go: https://github.com/ollama/ollama/pull/6279/files#r1862764190